datadog_checks_base/datadog_checks/base/checks/win/wmi/__init__.py

```diff
@@ -62,11 +62,12 @@ def _format_tag_query(self, sampler, wmi_obj, tag_query):
             )
             raise
         except TypeError:
+            wmi_property = tag_query[0]
+            wmi_class = sampler.class_name
             self.log.error(
-                u"Incorrect 'link source property' in `tag_queries` parameter:"
-                " `{wmi_property}` is not a property of `{wmi_class}`".format(
-                    wmi_property=tag_query[0], wmi_class=sampler.class_name
-                )
+                u"Incorrect 'link source property' in `tag_queries` parameter: `%s` is not a property of `%s`",
```

Comment:
We've got a bunch of `u"strings"` for log messages here, and probably in other places of the codebase as well. I don't *think* that's required even on Python 2, is it? (Eg. we've got many more checks dropping the `u` prefix for log messages.)

Reply:
Yeah AFAICT the logger works just fine with unicode and raw byte strings. But with formatting this can become dangerous when we mix unicode and raw string. Here in that case, if `wmi_property` or `wmi_class` is a non-ascii byte string, the formatting will fail with a UnicodeDecodeError. The opposite is as bad when you try to insert unicode strings in a raw string.

What I've seen here and there in the codebase is to pass parameters like `wmi_property` in our `to_string` method, the output is always a raw string on Py2 and a unicode string on Py3. And then we can do without the leading `u` and we will prevent mixing raw string and unicode.

In that case, your comment raises the point that if `wmi_property` or `wmi_class` can contain non-ascii code, this line will raise an Exception. Not sure how much this is a problem but it's not necessarily related to this PR